UIKitTemplate/UIKitTemplate/CustomUIButton.swift

```diff
@@ -0,0 +1,36 @@
+// CustomUIButton.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Кастомная кнопка для главного экрана и экрана выбора прожарки зерен
+struct CustomUIButton {
```

Comment:
```suggestion
struct CustomUIButton {
```
Custom это стоп слово как и Model